bin/reth/src/db/snapshots/mod.rs

```diff
@@ -90,15 +108,16 @@ impl Command {
         if self.only_bench || self.bench {
             for ((mode, compression), phf) in all_combinations {
                 match mode {
-                    Snapshots::Headers => self.bench_headers_snapshot(
+                    SnapshotSegment::Headers => self.bench_headers_snapshot(
                         db_path,
                         log_level,
                         chain.clone(),
                         *compression,
+                        InclusionFilter::Cuckoo,
```

Comment:
maybe we just replace this with another arg and add it to the `all_combinations`?